### DISCUSSION TITLE
National Football League's Stadiums

### DISCUSSION THREAD

**Discussion by emilyphantastic** [jakobzhao/geog495#64] [Final Project]
Link: https://emilyphantastic.github.io/Final-Web-GIS-App/
GitHub: https://github.com/emilyphantastic/Final-Web-GIS-App
Desc: This application shows all 30 of the football stadiums the National Football League (NFL) teams play in. There are 32 teams though, there are two teams in LA which share a stadium and two teams in New York which shares a stadium. This application is intended to allow users to see which states has an NFL team and how many. It also displays brief information of each stadium. Each stadium is also shown matched with it's corresponding football team. This application would be great for users who would like to learn about which state has an NFL team is and where professional football stadiums are located. I hope you enjoy! P.S. Go Hawks!
![image](https://user-images.githubusercontent.com/59709391/145533228-68eb727b-b5af-4f13-a289-7b75ddea4d9e.png)

**Comment by alexanderlyu** [comment]
The markers and the sidebar are really well made. I like the way that the information is displayed. However I think that the base map could customized more into something that makes the markers pop out more.

**Comment by ebar23** [comment]
I think that this is a really cool map layout. To be able to see geographically where all the NFL teams play and the way that the league is laid out. I would have to agree with Alexander that a switch of base map might work better especially with the NFL logos as your icons. Because the logos are .png's you can see straight through them and some of the map features kind of blend in with the logo.I think that a dark themed map would look good instead, but that's just a suggestion. Great job overall! Looks great :)

**Comment by ghost** [comment]
I really like how clean this map and features are, great work! I think the logo is a bit too transparent and blends a bit with your style choice, I would personally use a different style or a non-transparent image. But the size of the image looks perfect for this view!

**Comment by dldl1901s** [comment]
Your map is really cool. The sidebar slide is neat. The icon of the stadium also seems to have chosen very well. However, it would have been nice if there was more information such as the stadium address.

**Comment by meleakewubbie** [comment]
I like your topic since I'm a huge NFL fan. You chose a relevant icon by using the NFL logo on your map as stadium markers. I recommend removing city names that don't have an NFL stadium like Bakersfield, CA and Albany, NY etc.
